File: Modelo/Devolucion.cs
using System;

namespace punto_de_venta_C_.Modelo
{
    public class Devolucion
    {
        private int id;
        private int ventaId;
        private Producto producto;
        private double cantidad;
        private string motivo;
        private DateTime fecha;
        private string cajero;

        public Devolucion(int id, int ventaId, Producto producto, double cantidad, string motivo, DateTime fecha, string cajero)
        {
            this.id = id;
            this.ventaId = ventaId;
            this.producto = producto;
            this.cantidad = cantidad;
            this.motivo = motivo;
            this.fecha = fecha;
            this.cajero = cajero;
        }

        public int getId() { return id; }
        public void setId(int id) { this.id = id; }
        public int getVentaId() { return ventaId; }
        public Producto getProducto() { return producto; }
        public double getCantidad() { return cantidad; }
        public string getMotivo() { return motivo; }
        public DateTime getFecha() { return fecha; }
        public string getCajero() { return cajero; }

        public double getMontoDevuelto()
        {
            return Math.Round(producto.getPrecio() * cantidad * 100.0) / 100.0;
        }
    }
}

File: Modelo/Devolucion.py
class Devolucion:
    def __init__(self):
        self._id = 0
        self._ventaId = 0
        self._producto = None
        self._cantidad = 0.0
        self._motivo = ''
        self._fecha = None
        self._cajero = ''

    def getId(self):
        return self._id

    def setId(self, id):
        self._id = id

    def getVentaId(self):
        return self._ventaId

    def setVentaId(self, ventaId):
        self._ventaId = ventaId

    def getProducto(self):
        return self._producto

    def setProducto(self, producto):
        self._producto = producto

    def getCantidad(self):
        return self._cantidad

    def setCantidad(self, cantidad):
        self._cantidad = cantidad

    def getMotivo(self):
        return self._motivo

    def setMotivo(self, motivo):
        self._motivo = motivo

    def getFecha(self):
        return self._fecha

    def setFecha(self, fecha):
        self._fecha = fecha

    def getCajero(self):
        return self._cajero

    def setCajero(self, cajero):
        self._cajero = cajero
